application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -154,15 +154,13 @@ Failed to send a reminder with (authorID '{}') skipping it. This can be due to a
 
         int failureAttempts = pendingReminder.getFailureAttempts() + 1;
         Instant remindAt = Instant.now().plus(1, ChronoUnit.MINUTES);
-        database.write(context -> context.newRecord(PENDING_REMINDERS)
-            .setId(pendingReminder.getId())
-            .setCreatedAt(Instant.from(pendingReminder.getCreatedAt()))
-            .setGuildId(pendingReminder.getGuildId())
-            .setChannelId(pendingReminder.getChannelId())
-            .setAuthorId(pendingReminder.getAuthorId())
-            .setRemindAt(remindAt)
-            .setContent(pendingReminder.getContent())
-            .setFailureAttempts(failureAttempts)
-            .insert());
+        database.write(context -> {
+            pendingReminder.setRemindAt(remindAt);
+            pendingReminder.setFailureAttempts(failureAttempts);
+            pendingReminder.insert();
+        });
```

Comment:
to signal that you dont use `context`, you can call it `any` instead of `context` (in other languages people also like to use `_`, but thats not possible in java)

Reply:
Is it just an FYI or a suggestion to be added?

Reply:
that is a suggestion to be added, when i read the name any, ik that it is not used inside. So u should always have good naming